packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx

```diff
@@ -433,7 +433,13 @@ const DashboardChartTileMain: FC<DashboardChartTileMainProps> = (props) => {
                 isLoading={isLoading || isLoadingExplore}
                 extraMenuItems={
                     savedChartUuid !== null &&
-                    user.data?.ability?.can('manage', 'Explore') && (
+                    user.data?.ability?.can(
+                        'manage',
+                        subject('Explore', {
+                            organizationUuid: user.data?.organizationUuid,
```

Comment:
mm, ok, so this check organizationUuid is basically checking nothing

This is checking that user.orgUuid == user.orgUuid . 
What we should check here is if `organizationUuid: savedChart.orgUuid `

I can elaborate more if you don't see the difference

if you can't get the info or it is not available, I'd rather remove this from the subject

Reply:
this is the ability definition https://github.com/lightdash/lightdash/blob/main/packages/common/src/authorization/organizationMemberAbility.ts#L62

Reply:
This is a good "frontend" example about how to do this https://github.com/lightdash/lightdash/blob/main/packages/frontend/src/hooks/validation/useValidation.ts#L121

Reply:
Done